kong/db/migrations/core/015_280_to_300.lua

```diff
@@ -0,0 +1,19 @@
+return {
```

Comment:
All the db related changes are only needed for the control plane, right? If I were to connect my dp only to koko, I wouldn't need to port those code to the ee repo?

Reply:
right.  i think DP (or non-hybrid) never instantiates those Node entities.